### ISSUE TITLE
Configurable waiting time for initial SQL HealthCheck check

### ISSUE THREAD

**Issue by scrocquesel** [Azure/azure-service-bus-emulator-installer#15]
We are using test containers to wrap sql server and the emulator. This ensure the emulator is started only when the sql container is ready. Waiting 15 seconds is way too long in unit testing.

**Comment by Saglodha** [comment]
Thanks for your feedback @scrocquesel. We'll investigate and keep this issue open for further progress updates.

**Comment by mdelapenya** [comment]
Hi from the Testcontainers community 👋 ! Go maintainer here.

I'm adding the servicebus emulator module to the Go implementation and found this issue. From testcontainers standpoint, we are adding a dependency on the MSSQL container, which in the library code means the module will depend on the existing MSSQL testcontainers Go module to start up the MSSQL container, which already includes mechanisms to detect its readiness, so I'd advocate for enabling a way to disable the startup check (with an env var?) so that the TC modules can directly use that approach to simplify the integration testing story with the emulator.

Wdyt?

**Comment by Saglodha** [comment]
Hi all, 

Thanks for sharing your feedback. Happy to share that the latest version of Service Bus emulator provides user configurable health check interval for SQL. You can find detailed release notes here - https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-emulator-whats-new#latest-version-112 

Kindly reach out to us in case you have additional feedback or concerns.

**Comment by krishankumar95** [comment]
> Good to know it. Thanks! Then using the `SQL_WAIT_INTERVAL` environment variable would make it, right?

Yep; this variable can be used to tweak the initial wait interval for SQL Health-check. The value is delay in seconds prior to the check.

**Comment by mdelapenya** [comment]
Works like a charm! Thanks for this!
